website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,64 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### OIDC auth method attributes
+
+The OIDC auth method has the following additional attributes:
+
+- `account_claim_maps` (optional list) These are a map from custom
+  claims to the standard claims of sub, name and email. These maps are
+  represented as key=value where the key equals the Provider from-claim and the
+  value equals the Boundary to-claim. For example "oid=sub". May be specified
+  multiple times for different to-claims.
+
+- `allowed_audiences` (optional list) Audiences for which the provider
+  responses will be allowed
+
+- `api_url_prefix` (required) The API prefix to use when generating callback URLs
+  for the provider. Should be set to an address at which the provider can reach
+  back to the controller.
+
+- `callback_url` (output read-only) The URL that should be provided to the IdP
+  for callbacks.
+
+- `claims_scopes` (optional list) The claims scope requested. May be specified
+  multiple times.
+
+- `client_id` (required) The OAuth 2.0 Client Identifier this auth method should
+  use with the provider.
+
+- `client-secret` (required) The corresponding client secret.
+  
+- `client_secret_hmac` (output read-only) The HMAC of the client secret returned
+  by the Boundary controller, which is used for comparison after initial setting
+  of the value.
+
+- `description` - (optional)
+
+- `disable_discovered_config_validation` (optional) Disables validation logic
+  ensuring that the OIDC provider's information from its discovery endpoint
+  matches the information here. The validation is only performed at create or
+  update time.
+
+- `idp_ca_certs` - (optional) PEM-encoded X.509 CA certificate that can be used
+  as trust anchors when connecting to an OIDC provider. May be specified
+  multiple times.
+
+- `issuer` - (required) The provider's Issuer URL. which must match the issuer
+  field in generated tokens.
+
+- `max_age` (optional) The max age to provide to the provider, indicating how
+  much time is allowed to have passed since the last authentication before the
+  user is challenged again. A value of 0 sets an immediate requirement for all
+  users to reauthenticate, and an unset maxAge results in a Terraform value of
+  -1 and the default TTL of the chosen OIDC will be used.
+
+- `name` (optional) The auth-method name.
```

Comment:
Same as `description`, I think we should delete the entry for `name` here, since it's listed at the top of this page as an attribute that all auth methods have in common.